index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" content="width=device-width, initial-scale=1" name="viewport"/>
```

Comment:
I had never run into this issue before of our media query not working without these attributes. WordPress probably took care of that for me :)